packages/monitors/package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "bin": "index.js",
   "scripts": {
-    "test": "hardhat test"
+    "test": "hardhat test --network hardhat"
```

Comment:
Just curious -- why is the network required here but not in the others?

Reply:
I added it on all the `hardhat test` parts to ensure that we are using the hardhat network and there is no confusion about the environment that hardhat test is running within.